pkg/postgres/walker/walker.go

```diff
@@ -353,7 +353,7 @@ func typeIsEnum(typ reflect.Type) bool {
 func handleStruct(ctx walkerContext, schema *Schema, original reflect.Type) {
 	for i := 0; i < original.NumField(); i++ {
 		structField := original.Field(i)
-		if strings.HasPrefix(structField.Name, "XXX") {
+		if structField.Tag.Get("protobuf") == "" && structField.Tag.Get("protobuf_oneof") == "" {
```

Comment:
How about creating a func for this? Maybe we can extract all `XXX` checks into functions first and then just change their implementation